os/include/semaphore.h

```diff
@@ -200,6 +200,7 @@ int sem_wait(FAR sem_t *sem);
  * @since Tizen RT v1.0
  */
 int sem_timedwait(FAR sem_t *sem, FAR const struct timespec *abstime);
+int sem_tickwait(FAR sem_t *sem, systime_t start, uint32_t delay);
```

Comment:
No. This isn't standard API. You must put it in os/include/tinyara/semaphore.h. Not in here.